Assets/Thirdweb/Core/Scripts/AccountAbstraction/Core/BundlerClient.cs

```diff
@@ -55,8 +55,9 @@ private static async Task<RpcResponseMessage> BundlerRequest(string url, string
             var httpRequestMessage = new HttpRequestMessage(HttpMethod.Post, url) { Content = new StringContent(requestMessageJson, System.Text.Encoding.UTF8, "application/json") };
             if (new Uri(url).Host.EndsWith(".thirdweb.com"))
             {
-                httpRequestMessage.Headers.Add("x-sdk-name", "UnitySDK");
-                httpRequestMessage.Headers.Add("x-sdk-platform", Utils.GetRuntimePlatform());
+                httpRequestMessage.Headers.Add("x-sdk-name", "unity");
```

Comment:
can you leave `UnitySDK` as the `x-sdk-name`? easier to track with past data

Reply:
fixed